index.js

```diff
@@ -13,6 +14,14 @@ class GEmojiElement extends HTMLElement {
     }
   }
 
+  get tone(): number {
+    return this.hasAttribute('tone') ? parseInt(this.getAttribute('tone'), 10) : 0
+  }
+
+  set tone(modifier: number) {
```

Comment:
Perhaps an overkill 🙈  but we can potentially tighten this up a little with a `MODIFIERS` type so `emoji.tone = 999` fails type check.

```suggestion
  set tone(modifier: MODIFIERS) {
```